src/gui/MixerChannelView.cpp

```diff
@@ -128,8 +128,8 @@ namespace lmms::gui
         m_effectRackView->setFixedWidth(EffectRackView::DEFAULT_WIDTH);
 
         auto mainLayout = new QVBoxLayout{this};
-        mainLayout->setContentsMargins(0, 0, 0, 0);
-        mainLayout->setSpacing(0);
+        mainLayout->setContentsMargins(3, 3, 3, 3);
```

Comment:
I believe 4 would be a good value and for setSpacing, try 2.

Reply:
I am changing the horizontal margins to 4 and the vertical margins to match the spacing specified. A horizontal margin of 4 seems to work well in terms of keeping enough whitespace (3 arguably works too, but 4 might be a bit better). A vertical margin that matches how much we space the child widgets in the `QVBoxLayout` makes sense to me: if we are spacing the child widgets vertically by 2, it would make sense to use that same amount of space for our vertical margins at the edges of the layout.

Reply:
Nevermind, it seems like the margin space and the amount of spacing between widgets aren't exactly the same thing.